backend/app/controllers/api/v1/investors_controller.rb

```diff
@@ -4,6 +4,7 @@ class InvestorsController < BaseController
       include API::Pagination
 
       before_action :fetch_investor, only: :show
+      around_action :set_locale, only: [:show]
```

Comment:
I have seen that this repeats here soo often and only difference in method implementation is how you retrieve record language. Maybe you could define `proc` (https://guides.rubyonrails.org/action_controller_overview.html#other-ways-to-use-filters) here and reuse `set_locale` everywhere:
```
around_action { |_controller, action| set_locale(@investor&.account&.language, &action) }, only: [:show]
```

Reply:
OK, actually I started wondering whether I need this at all, so I was going to remove the around_action, but this is better